netci.groovy

```diff
@@ -0,0 +1,23 @@
+import jobs.generation.Utilities
+
+def project = GithubProject
+def branch = GithubBranchName
+def isPR = true
+def hostOS = "Ubuntu16.04"
+def machineLabel = 'latest-or-auto-docker'
+def distroList = ['alpine', 'centos', 'debian', 'fedora', 'opensuse', 'ubuntu']
+
+distroList.each { distro ->
+    def newJobName = Utilities.getFullJobName(project, "${distro}", isPR)
+
+    def newJob = job(newJobName) {
+        steps {
+            shell("docker build -t runner --pull -f ./Dockerfile.linux.runner . ")
```

Comment:
Nit - there is an unnecessary space after the ending `.`

Reply:
Fixed.